src/main/java/seedu/address/logic/Messages.java

```diff
@@ -67,4 +68,15 @@ public static String format(Project project) {
         builder.substring(0, builder.lastIndexOf(", ") == -1 ? builder.length() : builder.lastIndexOf(", "));
         return builder.toString();
     }
+
+    /**
+     * Formats the {@code Task} for display to the user.
+     */
+    public static String format(Task task) {
+        final StringBuilder builder = new StringBuilder();
+        builder.append(task.getName());
+        builder.append("; Deadline: " + task.getDeadline().toString());
```

Comment:
Should the deadline here be formatted? I believe that currently, it may result in a string that is not very user-friendly.

Reply:
Have modified the code on my local repo to display as such now. (see below screenshot): Will be fixed in a future PR

<img width="746" alt="image" src="https://github.com/AY2324S1-CS2103T-T08-3/tp/assets/64006665/292b465b-720a-4e2a-998e-1f9f6c2384d8">